lib/index.js

```diff
@@ -34,6 +34,8 @@ var CoverallsReporter = function(rootConfig, helper, logger) {
         filepath = path.resolve(rootConfig.basePath, reporter.dir);
         break;
       }
+    } else {
+     throw new Error("coverage reporter should precede coveralls");
```

Comment:
This doesn't look right. Why not just throw if `rootConfig.coverageReporter` is undefined?